src/Microsoft.Performance.SDK.Runtime/Options/Serialization/Loading/StreamPluginOptionsLoader.cs

```diff
@@ -42,8 +51,9 @@ public async Task<PluginOptionsDto> TryLoadAsync()
         {
             return await JsonSerializer.DeserializeAsync<PluginOptionsDto>(stream, jsonSerializerOptions);
         }
-        catch
+        catch (Exception e)
         {
+            this.logger?.Error(e, "Failed to load plugin options from stream.");
```

Comment:
>this.logger?.Error(e, "Failed to load plugin options from stream."); [](http://example.com/codeflow?start=12&length=68)

it might be nice to have some stream identifier here - such as file name for the FilePluginOptionsLoader.